client/components/SomeComponent/__test__/Index.test.tsx

```diff
@@ -4,15 +4,15 @@
 
 import React from 'react';
 import { render } from '@testing-library/react';
-import { ThemeProvider } from 'styled-components';
+import { ThemeProvider } from '@material-ui/core/styles';
 import { SomeComponent } from 'client/components';
-import { themeObject } from 'styles/theme';
+import theme from 'styles/theme';
 
 describe('SomeComponent', () => {
   it('should render text', () => {
     const { container } = render(
-      <ThemeProvider theme={themeObject}>
-        <SomeComponent />
+      <ThemeProvider theme={theme}>
+        <SomeComponent />,
```

Comment:
remove comma from jsx

Reply:
Actually not, this is a param in the render() function. And we have `"trailingComma": "all"`, so prettier adds it automatically

Reply:
That shouldn't be the case because ThemeProvider is still unclosed be, so it'll render in the dom unless I'm missing something? There's a comma on the next line for ThemeProvider which makes sense since it's the end of the jsx block but that shouldn't be the case here

Reply:
Crap, there are two commas, only noticed one in this small snippet 🙈